lms/djangoapps/mobile_api/video_outlines/serializers.py

```diff
@@ -196,14 +199,19 @@ def video_summary(video_profiles, course_id, video_descriptor, request, local_ca
             if default_encoded_video:
                 break
 
+    video_url = None
```

Comment:
The if statement below is well defined, so `video_url = None` should be redundant.

Reply:
Fixed https://github.com/edx/edx-platform/pull/16590/commits/879c979d49eab7c8f2773fe2f13c031234cc3e72#diff-de0e3088a167332527d38a99af73ea19R196